src/test/scala/uk/gov/hmrc/test/ui/pages/section1/DeclarationChoicePage.scala

```diff
@@ -16,14 +16,20 @@
 
 package uk.gov.hmrc.test.ui.pages.section1
 
-import uk.gov.hmrc.test.ui.pages.section1.DetailKeys.DeclarationType
+import uk.gov.hmrc.test.ui.pages.base.Constants._
+import uk.gov.hmrc.test.ui.pages.base.TariffLinks.{decType, decTypeCL}
 import uk.gov.hmrc.test.ui.pages.base.{BasePage, Detail}
+import uk.gov.hmrc.test.ui.pages.section1.DetailKeys.DeclarationType
 
 object DeclarationChoicePage extends BasePage {
 
   val backButtonHref: String = StandardOrOtherPage.path
```

Comment:
Unless you assign a static String, it's safer use always a `def`, since we are referring to the url of another page, which maybe will become dynamic in the future